catalog/app/containers/Admin/RolesAndPolicies/Policies.tsx

```diff
@@ -0,0 +1,735 @@
+import * as FF from 'final-form'
+import * as IO from 'io-ts'
+import * as R from 'ramda'
+import * as React from 'react'
+import * as RF from 'react-final-form'
+import * as urql from 'urql'
+import * as M from '@material-ui/core'
+
+import * as Notifications from 'containers/Notifications'
+import * as Model from 'model'
+import * as Dialogs from 'utils/Dialogs'
+import type FormSpec from 'utils/FormSpec'
+import assertNever from 'utils/assertNever'
+import { mkFormError, mapInputErrors } from 'utils/formTools'
+import * as Types from 'utils/types'
+import validate, * as validators from 'utils/validators'
+
+import * as Form from '../RFForm'
+import * as Table from '../Table'
+
+import AssociatedRoles from './AssociatedRoles'
+import BucketsPermissions from './BucketsPermissions'
+import { getArnLink } from './shared'
+
+import POLICIES_QUERY from './gql/Policies.generated'
+import POLICY_CREATE_MANAGED_MUTATION from './gql/PolicyCreateManaged.generated'
+import POLICY_CREATE_UNMANAGED_MUTATION from './gql/PolicyCreateUnmanaged.generated'
+import POLICY_UPDATE_MANAGED_MUTATION from './gql/PolicyUpdateManaged.generated'
+import POLICY_UPDATE_UNMANAGED_MUTATION from './gql/PolicyUpdateUnmanaged.generated'
+import POLICY_DELETE_MUTATION from './gql/PolicyDelete.generated'
+import { BucketPermissionSelectionFragment as BucketPermission } from './gql/BucketPermissionSelection.generated'
+import { PolicySelectionFragment as Policy } from './gql/PolicySelection.generated'
+
+const validateNonEmptyString: FF.FieldValidator<any> = validate(
+  'nonEmptyString',
+  validators.matches(/\S/),
+)
+
+const columns = [
+  {
+    id: 'title',
+    label: 'Title',
+    getValue: R.prop('title'),
+    props: { component: 'th', scope: 'row' },
+  },
+  {
+    id: 'source',
+    label: 'Source',
+    getValue: (p: Policy) => p.managed,
+    getDisplay: (value: boolean) =>
+      value ? (
+        <abbr title="This IAM policy is created and managed by Quilt">Quilt</abbr>
```

Comment:
The more right way to add a dotted line is to use CSS. `<abbr>` has semantic meaning

Reply:
abbr here isnt used primarily for a dotted line, but more importantly for the tooltip that explains the meaning of the type of a role, so i think it's kinda correct semantically (to some extent)